src/main/java/com/autotune/common/data/system/info/device/accelerator/metadata/AcceleratorMetaDataService.java

```diff
@@ -0,0 +1,78 @@
+package com.autotune.common.data.system.info.device.accelerator.metadata;
+
+
+
+import com.autotune.analyzer.utils.AnalyzerConstants;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AcceleratorMetaDataService {
+    private static Map<String, List<AcceleratorProfile>> acceleratorProfilesMap;
+    private static AcceleratorMetaDataService acceleratorMetaDataService = null;
+
+    private AcceleratorMetaDataService(){
+        acceleratorProfilesMap = new HashMap<>();
+        initializeAcceleratorProfiles();
+    }
+
+    private static void initializeAcceleratorProfiles() {
+        List<AcceleratorProfile> commonProfiles = new ArrayList<>();
+        // IMPORTANT: Add it in the ascending order according to GPU Core and Memory Units as we will break the loop upon getting the right one
+        commonProfiles.add(new AcceleratorProfile("1g.10gb", 1.0 / 8, 1.0 / 7, 7));
+        commonProfiles.add(new AcceleratorProfile("1g.20gb", 1.0 / 4, 1.0 / 7, 4));
+        commonProfiles.add(new AcceleratorProfile("2g.20gb", 2.0 / 8, 2.0 / 7, 3));
+        commonProfiles.add(new AcceleratorProfile("3g.40gb", 4.0 / 8, 3.0 / 7, 2));
+        commonProfiles.add(new AcceleratorProfile("4g.40gb", 4.0 / 8, 4.0 / 7, 1));
+        commonProfiles.add(new AcceleratorProfile("7g.80gb", 1.0, 1.0, 1));
```

Comment:
Create constants for all of this. Same for the ones below

Reply:
Created the constants for the profile names, excluded the constants for fractions. Should I create constants for fractions as well?